### DISCUSSION TITLE
How to get Content Items from Keyword in Global Search on content item.

### DISCUSSION THREAD

**Discussion by Syed-Hasnain-Askari** [OrchardCMS/OrchardCore#12277] [Q&A]
Folks,

I create query see below:

![image](https://user-images.githubusercontent.com/47834896/186954818-cda8b293-bf16-4cf6-afbc-bbd389061fcd.png)

detail of query below:

![image](https://user-images.githubusercontent.com/47834896/186954911-5683241b-e77b-4622-af4f-e626f141dbb2.png)

I'm working on Orchard Lucene Search, I want to search content Items based on my keyword. for example, there's is 8 Content Item of "Prime" so when I type Prime so all the Prime will be show.

This is my query which is work fine.
`{
  "query": {
    "bool": {
      "must" : {
          "term" : { "Content.ContentItem.ContentType" : "Software" }
      },
      "filter": [
        { "term": { "Content.ContentItem.Published" : "true" }},
        { "wildcard": { "Content.ContentItem.DisplayText" : "Prime*" }}
      ]
    }
  }
}`
![image](https://user-images.githubusercontent.com/47834896/186918597-e3269498-9625-4488-9cc8-40a267cae4de.png)

how to override the Orchard Lucene Search default behavior based on above structure.

**Comment by Skrypt** [comment]
closing as duplicate
